explorer/client/src/pages/object-result/ObjectLoaded.tsx

```diff
@@ -183,6 +183,12 @@ function ObjectLoaded({ data }: { data: DataType }) {
         .filter(([key, _]) => !/name/i.test(key))
         .filter(([_, value]) => checkIsPropertyType(value));
 
+    const descriptionTitle =
+        data.objType === 'Move Package' ? 'Package Description' : 'Description';
+
+    const detailsTitle =
+        data.objType === 'Move Package' ? 'Source Code' : 'Properties';
```

Comment:
it would be a lie to call this source code

Reply:
"Disassembled bytecode" is one good option for a descriptor.